bin/rl/rl.cpp

```diff
@@ -228,7 +228,7 @@ TARGETINFO TargetInfo[] = {
    "sh5c",       FALSE, FALSE, FALSE, "sh5sim",       "libcsim.lib", "x86asm",
    "sh5m",       FALSE, FALSE, FALSE, "sh5sim",       "libcsim.lib", "x86asm",
    "ia64",       FALSE, TRUE,  TRUE,  NULL,           NULL, "x86asm",
-   "amd64",      FALSE, TRUE,  TRUE,  NULL,           NULL, "x86asm",
```

Comment:
I think we should keep both, that way it will work regardless of which we use. I imagine `ia64` is just an alias for `amd64` at this time, we should have the same behavior to map `amd64` to `x64` to keep all existing tools working